linera-core/src/local_node.rs

```diff
@@ -294,35 +276,6 @@ where
         Ok(response)
     }
 
-    /// Obtains the certificate containing the specified message.
-    #[instrument(level = "trace", skip(self))]
-    pub async fn certificate_for_block(
```

Comment:
I can imagine this coming in handy - especially when improving the performance of syncing parent chain. We want to avoid downloading everything so I was thinking that we would download certificates for "latest" block and go from there.

Reply:
That's true. We can always bring it back if needed. But I'm not a fan of keeping around unused code in case we need it.